### DISCUSSION TITLE
Mixing paths with Jinja2 variables

### DISCUSSION THREAD

**Discussion by NFanoe** [mondeja/mkdocs-include-markdown-plugin#229] [Q&A]
I seem to be unable to do something like this:
`{% include-markdown '{{customer|lower}}/doc/solution/filename.md' %}`
Is that so? Would be great to be able to add include links that work in multiple doc sets.

**Comment by mondeja** [comment]
If your problem is related with Jinja2, use different opening and closing tags, see https://github.com/mondeja/mkdocs-include-markdown-plugin?tab=readme-ov-file#opening_tag-and-closing_tag

**Reply by DeNelo** [reply]
I'm afraid that doesn't cut it. I have changed the opening and closing tags as you suggested. But this:

```
---
includefile: dok1.md
---
  
# Welcome to MkDocs

For full documentation visit [mkdocs.org](https://www.mkdocs.org).

## Commands

* `mkdocs new [dir-name]` - Create a new project.
* `mkdocs serve` - Start the live-reloading docs server.
* `mkdocs build` - Build the documentation site.
* `mkdocs -h` - Print help message and exit.

{{includefile}}

{!
    include-markdown "{{includefile}}"
    heading-offset=1
!}
```
-does not work (build error on mkdocs serve).
It would be a really useful thing for us!

**Reply by DeNelo** [reply]
What if the macros got expanded _before_ include-markdown was effected ..?

**Reply by mondeja** [reply]
PRs welcome
